### PR TITLE
Adding requests query

### LINKED ISSUE #335: RA/UM - build Requests report query
https://issues.folio.org/browse/UXPROD-2029

### PR DESCRIPTION
**Pull request by amzoss**
will close #335

### CODE REVIEW

File: sql/report_queries/requests/requests.sql
Comment (adoerrer) [inline sql/report_queries/requests/requests.sql:78]: Is there a reason to add the schema public here?

**Review by adoerrer — COMMENTED** [review]
Hi @amzoss, overall this looks good but is there a database where this might be tested against?

**Comment by amzoss** [comment]
> Hi @amzoss, overall this looks good but is there a database where this might be tested against?

Sorry, yes! I think it was working on Tuesday (2020-04-06)

File: sql/report_queries/requests/requests.sql
Comment (amzoss) [reply sql/report_queries/requests/requests.sql:78]: No, I'm not sure why I added that, should work fine without it

**Comment by adoerrer** [comment]
> > Hi @amzoss, overall this looks good but is there a database where this might be tested against?
> 
> Sorry, yes! I think it was working on Tuesday (2020-04-06)

Thanks! Looks good to go.

**Review by adoerrer — APPROVED** [review]

**Review by nassibnassar — CHANGES_REQUESTED** [review]
Is this missing documentation?

**Comment by amzoss** [comment]
> Is this missing documentation?

I thought the query work needed to be done before freeze but documentation could be added after, so we haven't been prioritizing it for this week. But we can wait to approve this until docs are done, if that's better. 

We also have another edit to this query, so we can also wait for that before merging.

**Comment by nassibnassar** [comment]
I would prefer in the future that we combine them into a single PR.

**Review by nassibnassar — APPROVED** [review]

**Pull request merged**
